EIPS/eip-1.md

```diff
@@ -266,6 +286,79 @@ Many EIPs are written and maintained by developers with write access to the Ethe
+
+The following is the specification and definition of the Hard Fork process for Ethereum, governed by the AllCoreDevs Community Calls.
+
+#### Specification
+A Meta EIP should be created and merged as a Draft in preparation for the hardfork.
+
+This EIP should contain:
+ - the desired codename of the hard fork,
+ - activation block number once decided
+ - a timeline section
+ - an EIPs to include section
+ - the Requires header should point to the previous hard fork meta EIP.
+
+The draft shall be updated with summaries of the decisions around the hard fork.
+
+##### Timeline
+Once a timeline with key dates is agreed upon for other crucial dates. The basic outline of a hardfork timeline should include:
+
+- Projected date for testnet network upgrade
+- Projected date for mainnet upgrade (the activation block number / projected date for this block)
+
+#### EIP Inclusion Process
+Anyone that wishes to propose a Core EIP for the hard fork should make a PR against the Meta EIP representing the hard fork. For an EIP to be considered it must follow the EIP centric model defined later in this document and must be published in at least `DRAFT` status. It enters the Proposed EIPs section, along with at least one person who is the champion for that EIP.
+
+EIPs can move along this process by participating in discussions on the “All Core Devs Meetings”:
+- If accepted for a hard fork, the EIP should be moved to the Accepted EIPs section.
+- Once the EIPs in the Accepted EIPs section have successfully launched on a testnet roll out, they are moved to the Included EIPs section.
+
+The Meta EIP representing the hard fork should move in to the Accepted state once the changes are frozen (i.e. all referenced EIPs are in the Accepted state) and in to the Final state once the hard fork has been activated.
+
+### Ethereum Hardforks
+ - Genesis Block *block 1*
+ - [Homestead](https://eips.ethereum.org/EIPS/eip-606) *block 1_150_000*
+ - [Spurious Dragon](https://eips.ethereum.org/EIPS/eip-607) *block 2_675_000*
+ - [Byzantium](https://eips.ethereum.org/EIPS/eip-609) *block 4_370_000*
+ - [Constantinople](https://eips.ethereum.org/EIPS/eip-1013) *block 7_280_000*
+ - [Istanbul](https://eips.ethereum.org/EIPS/eip-1679) *block 9_069_000*
+ - [Muir Glacier](https://eips.ethereum.org/EIPS/eip-2387) *block 9_200_000*
+
+### EIP Centric Model
+
+The EIP centric forking model defines the approval process for any EIP to be included in a hardfork or network upgrade. 
+
+The pipeline for Core EIPs is as follows.
+
+`[ DRAFT ] -> [ ELLIGLE FOR INCLUSION ] -> [ IMPLEMENTATION ] -> [ TESTING ] -> [ ACCEPTED ] -> [ DEPLOYED ]`
+
+Note that this process is included within the EIP Flow, and so `LAST_CALL` is still observed as part of the higher-order process of EIP finalization.
```

Comment:
This is a bit unclear. Should we just add `LAST_CALL` after `[ ACCEPTED ]` on line 339?

Reply:
There is some nuance about where this information is tracked, but I am getting ahead of myself as that is relevant when we have a "staff editing section" in the EIP. Updated with LAST CALL. 👍